### ISSUE TITLE
How to set font size in TextField?

### ISSUE THREAD

**Issue by MarkLFT** [enisn/UraniumUI#273]
Is there a way to set the font size in the controls? Our forms have a lot of fields so was hoping to be able to use a smaller font.

**Comment by enisn** [comment]
You can try to resize labels in a single textfield like the following way:
```xml
<material:TextField Title="Email"
                    Text="{v:ValidationBinding Email}" 
                    Icon="{FontImage FontFamily=MaterialRegular, Glyph={x:Static m:MaterialRegular.Email}}">
                    <material:TextField.Resources>
                        <ResourceDictionary>
                            
                            <Style TargetType="Label">
                                <Setter Property="FontSize" Value="11" />
                            </Style>
                        </ResourceDictionary>
                    </material:TextField.Resources>
</material:TextField>
```

**Comment by pcchong2000** [comment]
Is there a way to set the font size of the TextField inside the control instead of TargetType="Label"?  Thanks

**Comment by enisn** [comment]
There is not currently, it's a regular label inside TextField. So styling will work. 

We can add a dedicated property for that in the next version

**Comment by Visutronik** [comment]
Hi, is it now possible to change the font size of a TextField (text and title) or is it a feature for the next version?
It would great if it's possible to change the font size for all InputFiled Controls, because the text and title is a little bit to small on a tablet and in Windows.

**Comment by enisn** [comment]
Why did I closed this Issue I'm not sure. I can add those properties in the next version.

**Comment by enisn** [comment]
It's added and will be included in `v2.6.0-pre.2`

**Comment by yousufctec** [comment]
> You can try to resize labels in a single textfield like the following way:
> 
> ```
> <material:TextField Title="Email"
>                     Text="{v:ValidationBinding Email}" 
>                     Icon="{FontImage FontFamily=MaterialRegular, Glyph={x:Static m:MaterialRegular.Email}}">
>                     <material:TextField.Resources>
>                         <ResourceDictionary>
>                             
>                             <Style TargetType="Label">
>                                 <Setter Property="FontSize" Value="11" />
>                             </Style>
>                         </ResourceDictionary>
>                     </material:TextField.Resources>
> </material:TextField>
> ```

Will this work for EditorField? I am trying and it's not working. Appreciate any help.
